{{cookiecutter.app_name}}/Makefile

```diff
@@ -1,19 +1,47 @@
-.PHONY: env-setup env-teardown dev install-dependencies test
+include .env
+ifdef ENV
+include .env.$(ENV)
+endif
+
+.PHONY: env-setup env-teardown db/migrate db/rollback migratoin/create migration/status dev install-dependencies test wait-for-postgres
 
 env-setup:
 	docker-compose -f docker-compose.dev.yml up -d
 
 env-teardown:
 	docker-compose -f docker-compose.dev.yml down
 
-dev: env-setup
+db/migrate:
+	make wait-for-postgres
+	goose -dir database/migrations -table "migration_versions" postgres "$(DATABASE_URL)" up
+
+db/rollback:
```

Comment:
Shouldn't this also wait with `make wait-for-postgres`? `db/rollback` could be run at any time even as the first command.

Reply:
Good catch. Fixed in 2037ed85ccc8c81525ab43a5163a6d03c57a583c